debian/changelog

```diff
@@ -1,4 +1,4 @@
-sbank (0.0.2-1) unstable; urgency=low
+sbank (0.0.3-1) unstable; urgency=low
```

Comment:
I think you are supposed to add a new entry to the Debian changelog, as opposed to editing a previous entry. Not sure what practical effects this has though.